clang-tools-extra/test/clang-tidy/checkers/misc/const-correctness-values.cpp

```diff
@@ -1007,3 +1007,11 @@ template <typename T> void f() {
   x[T{}] = 3;
 }
 } // namespace gh127776_false_positive
+
+namespace gh132931_false_positive {
+using T = const int;
+void valid(int i) {
+  const int arr0[] = {1, 2, 3};
+  T arr1[] = {1, 2, 3};
```

Comment:
Shouldn't we get a warning here, since `arr1` is not used and can therefore be const?

Reply:
Ah sorry I see now `T` is `const int` :man_facepalming: